command.go

```diff
@@ -477,6 +483,37 @@ func (c *Command) Help() error {
 	return nil
 }
 
+// SuggestFunc returns suggestions for the provided typedName using either
+// the function set by SetSuggestFunc for this command, parent's or a default one.
+// When searching for a parent's function, it recursively checks towards the root
+// and returns the first one found. If none found, uses direct parent's default.
+func (c *Command) SuggestFunc(typedName string) string {
+	if c.DisableSuggestions {
+		return ""
+	}
+	if c.suggestFunc != nil {
+		return c.suggestFunc(typedName)
+	}
+	if c.HasParent() {
+		var getParentFunc func(*Command) func(string) string
+		getParentFunc = func(parent *Command) func(string) string {
```

Comment:
You can inline it, no?

```suggestion
		getParentFunc := func(parent *Command) func(string) string {
```

Reply:
No, have to define it separately because of the recursion.